encoder/src/main/java/com/pedro/encoder/input/gl/render/SimpleCameraRender.java

```diff
@@ -90,8 +90,9 @@ public void drawFrame(int width, int height, boolean keepAspectRatio, int mode,
     GlUtil.checkGlError("drawFrame start");
     surfaceTexture.getTransformMatrix(STMatrix);
 
-    if (mode == 2 || mode == 3) { //stream rotation is enabled
-      SizeCalculator.updateMatrix(rotation, width, height, isPreview, isPortrait, MVPMatrix);
+    if(mode == 2 || mode == 3) {
```

Comment:
Don't we want this logic here as well? Btw wrong formatting next to `if`

Reply:
Removed the if completely